cmd/explaintest/r/explain_easy.result

```diff
@@ -327,3 +327,29 @@ id	count	task	operator info
 Projection_3	10000.00	root	0
 └─TableReader_5	10000.00	root	data:TableScan_4
   └─TableScan_4	10000.00	cop	table:t, range:[-inf,+inf], keep order:false, stats:pseudo
+drop table if exists t;
+create table t(a bigint, b bigint, index idx(a, b));
+explain select * from t where a in (1, 2) and a in (1, 3);
+id	count	task	operator info
+IndexReader_9	10.00	root	index:IndexScan_8
+└─IndexScan_8	10.00	cop	table:t, index:a, b, range:[1,1], keep order:false, stats:pseudo
+explain select * from t where b in (1, 2) and b in (1, 3);
+id	count	task	operator info
+TableReader_7	10.00	root	data:Selection_6
+└─Selection_6	10.00	cop	in(test.t.b, 1, 2), in(test.t.b, 1, 3)
+  └─TableScan_5	10000.00	cop	table:t, range:[-inf,+inf], keep order:false, stats:pseudo
+explain select * from t where a = 1 and a = 1;
+id	count	task	operator info
+IndexReader_9	10.00	root	index:IndexScan_8
+└─IndexScan_8	10.00	cop	table:t, index:a, b, range:[1,1], keep order:false, stats:pseudo
+explain select * from t where a = 1 and a = 2;
+id	count	task	operator info
+TableDual_8	0.00	root	rows:0
+explain select * from t where b = 1 and b = 2;
+id	count	task	operator info
+TableDual_8	0.00	root	rows:0
+drop table if exists t;
+create table t(a bigint primary key);
+explain select * from t where a = 1 and a = 2;
+id	count	task	operator info
+TableDual_5	10000.00	root	rows:0
```

Comment:
the `count` field of the explain seems wrong, it should be `0.00` instead. Is is computed by the following code snippet in function `GetRowCountByIntColumnRanges` called by `Selectivity`:
```go
404     if coll.ColumnIsInvalid(sc, colID) {
405         if len(intRanges) == 0 {
406             return float64(coll.Count), nil
407         }
408         if intRanges[0].LowVal[0].Kind() == types.KindInt64 {
409             return getPseudoRowCountBySignedIntRanges(intRanges, float64(coll.Count)), nil
410         }
411         return getPseudoRowCountByUnsignedIntRanges(intRanges, float64(coll.Count)), nil
412     }
```

line 406 does not make sense, it should return `0, nil` when range is empty. Anyway, this should be resolved in another issue, and we can update this result later.

Reply:
issue for this problem: https://github.com/pingcap/tidb/issues/7582
PR to fix it: https://github.com/pingcap/tidb/pull/7583